conf/application.conf

```diff
@@ -175,7 +175,7 @@ self {
 }
 
 urls {
-  claimService = "/claim-for-reimbursement-of-import-duties/start"
+  claimService = "http://localhost:7500/claim-back-import-duty-vat/"
```

Comment:
do we need localhost explicitly? what happens when we go to dev/staging/qa? will it work with just '/'

Reply:
They have their own specific config files

Reply:
Environment specific config file